e2e/imgbuild/imgbuild.go

```diff
@@ -121,6 +121,12 @@ func (c *imgBuildTests) nonRootBuild(t *testing.T) {
 			buildSpec: "docker://busybox:latest",
 			sandbox:   false,
 		},
+		// #4517 - CentOS7 sandbox as non-root needs permissions forced
```

Comment:
Lets move this into `e2e/regression` since with the new work to build sandboxes in place within `master`, the `mv` currently occurring will no longer exist to trigger an error in this case.